### PR TITLE
RecoLocalMuon/CSCValidation: formatting fix for gcc 6.0 misleading-indentation warning

### PR DESCRIPTION
**Pull request by gartung**
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/RecoLocalMuon/CSCValidation/src/CSCValidation.cc:2594:15: warning: this 'for' clause does not guard... [-Wmisleading-indentation]
                for(unsigned int i=0;i<recIt->nStrips();i++)
               ^~~
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/RecoLocalMuon/CSCValidation/src/CSCValidation.cc:2600:17: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'for'
                 if(adc_3_3_sum > 100.0) {
                 ^~

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test
.. ?w=1 gives no diffs, btw

**Comment by gartung** [comment]
gcc 6.0 is picky about indentation of statements.

**Comment by gartung** [comment]
google "-Wmisleading-indentation"

**Comment by gartung** [comment]
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc600/CMSSW_8_1_X_2016-06-26-2300/RecoLocalMuon/CSCValidation

**Comment by gartung** [comment]
@Dr15Jones care to comment.

**Comment by Dr15Jones** [comment]
@gartung could you replace the tabs with spaces to match the original file? that would make it easier to see the changes.

**Comment by gartung** [comment]
The original had tabs. I will replace them with spaces.

File: RecoLocalMuon/CSCValidation/src/CSCValidation.cc [hunk 1]
```diff
@@ -2583,57 +2583,57 @@ void CSCValidation::doADCTiming(const CSCRecHit2DCollection& rechitcltn) {
...
               for(unsigned int i=0;i<recIt->nStrips();i++) 
-		for(unsigned int j=0;j<recIt->nTimeBins();j++)
-		  if(recIt->adcs(i,j)>adcmax) {
-		    adcmax=recIt->adcs(i,j); 
-		    binmx=j;
-		  }
-
-	      adc_3_3_sum=0.0;
-	      //well, this really only works for 3 strips in readout - not sure the right fix for general case
-              for(unsigned int i=0;i<recIt->nStrips();i++)
```
Comment (Dr15Jones) [inline RecoLocalMuon/CSCValidation/src/CSCValidation.cc]: Comment: this line which is in the original warning

File: RecoLocalMuon/CSCValidation/src/CSCValidation.cc [hunk 2]
```diff
@@ -2583,57 +2583,57 @@ void CSCValidation::doADCTiming(const CSCRecHit2DCollection& rechitcltn) {
...
-
-	      adc_3_3_sum=0.0;
-	      //well, this really only works for 3 strips in readout - not sure the right fix for general case
-              for(unsigned int i=0;i<recIt->nStrips();i++) 
-		for(unsigned int j=binmx-1;j<=binmx+1;j++) 
-		  adc_3_3_sum+=recIt->adcs(i,j);
-
-
-                // ADC weighted time bin
-                if(adc_3_3_sum > 100.0) {
```
Comment (Dr15Jones) [inline RecoLocalMuon/CSCValidation/src/CSCValidation.cc]: Comment: this is the line which the compiler warns is not in the loop (even though the indention can be misleading).

**Comment by Dr15Jones** [comment]
As long as the compiler no longer complains, this code looks functionally identical to the original and the original appears to be what the author intended.

**Comment by Dr15Jones** [comment]
please test

**Comment by cvuosalo** [comment]
+1

For #15008 3e0024298a7a885b3cd0974b2f4f7b013fa104dc

Cosmetic changes to CSCValidation.cc to enhance readability. There should be no change in the output from the compiler.

The code changes are satisfactory, and Jenkins tests against baseline CMSSW_8_1_X_2016-06-28-1100 show no significant differences, as expected.

**Comment by davidlange6** [comment]
+1

**Pull request merged**